codesign/bin/codesign.dart

```diff
@@ -71,14 +73,15 @@ Future<void> main(List<String> args) async {
       help: 'the Flutter engine commit revision for which Google Cloud Storage binary artifacts should be codesigned.',
     )
     ..addMultiOption(
-      kCodesignFilepathOption,
-      help: 'The zip file paths to be codesigned. Pass this option multiple'
-          'times to codesign multiple zip files',
-      valueHelp: 'darwin-x64/font-subset.zip',
+      kOptionalSwitch,
```

Comment:
This approach isn't scalable. Someone should be able to give a generic zip with the list of paths they want codesigned.

Reply:
umm right the current implementation signs through gcloud buckets. it takes in an artifact path (that exists on gcloud buckets), and signs all the binaries inside the zip. I might be misunderstanding, umm for the requirement of `Someone give a generic zip with the list of paths they want codesigned`, would the action item be taking in extra parameters for custom gcloud upload/download bucket paths? (or I can add a parameter to take in a custom local zip path, where the zip is located by local file system) 

Right, the current implementation accommodates the various gcloud paths of ios usb dependencies, like the unsigned bucket path '$bucketPrefix/ios-usb-dependencies/unsigned/$rawName/$commitHash/$from', or the artifact bucket path`ios-usb-dependencies/$rawName/$commitHash/$destination`. umm in terms of customized support, would it make sense for us to take the gcloud bucket paths as input parameters instead?

I might be wrong, based on Casey's idea, I am thinking if we take the gcloud bucket paths instead, we would still support the engine paths by default (cause there are like 32 such cloud paths prefix and would be hard to provide through command line). umm I am not sure about scalability, I thought we are signing engine artifacts and ios usb artifacts, or maybe we would like to sign others as well?